### ISSUE TITLE
accessibility: Trickle not focusing when autoscroll is off

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2077]
### Subject of the issue
Focus control is part of the autoscroll behaviour in trickle and so doesn't trigger when autoscroll is off but component has completed and the trickle button is clicked. 
[Reference](https://github.com/adaptlearning/adapt-contrib-trickle/blob/master/js/adapt-contrib-trickle.js#L103)

### Steps to reproduce
Turn off autoscroll on trickle.
Complete a component.
Click "Continue"
Focus doesn't shift to next component.

### Expected behaviour
Focus should move to header of next component.
Need to figure out if focus and tabbing can be independent of each other, such that the screen reader cursor can follow focus to the text but the text doesn't have to be tabbable.
See #2075

**Comment by oliverfoster** [comment]
done
